operators/src/source/gdal_source.rs

```diff
@@ -0,0 +1,456 @@
+use crate::util::Result;
+
+use std::path::{PathBuf};
+use gdal::raster::dataset::Dataset as GdalDataset;
+use gdal::raster::rasterband::RasterBand as GdalRasterBand;
+//use gdal::metadata::Metadata;
```

Comment:
We need to handle metadata. Its More a todo

Reply:
Yeah, I guess `// TODO:` it is. I'm just worried that these comments just stay in our codebase, since the IDE will just add a new import for you. With todos, we stay on track :+1: .